sycl/test-e2e/Graph/Explicit/buffer_copy_host2target_offset.cpp

```diff
@@ -0,0 +1,54 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s %}
+//
+// CHECK-NOT: LEAK
+
+// Tests adding buffer copy offset
+// -- Host to Target (write path) --  nodes
+// using the explicit API and submitting
+// the graph.
+// The second run is to check that there are no leaks reported with the embedded
+// ZE_DEBUG=4 testing capability.
+
+#include "../graph_common.hpp"
+
+int main() {
+  queue Queue;
+
+  using T = int;
+
+  int Offset = 100;
```

Comment:
Might be better to calculate the offset based on the size to prevent potential issues if we change the size in the future to a value which breaks this.

Could also define the offsets in the common header alongside `Size` so we can remove that code from all tests that use them, and also make updating them collectively easier.